weldx/tests/asdf_tests/test_asdf_validators.py

```diff
@@ -163,6 +164,8 @@ def test_shape_validation_error_exception(shape, exp, err):
         assert _val(shape, exp)
 
 
+@pytest.mark.parametrize("copy_arrays", [True, False])
+@pytest.mark.parametrize("lazy_load", [True, False])
```

Comment:
we can remove this before merging @marscher

Reply:
Why, didn't this cover the ndarray/NDArrayType pair?

Reply:
Ah I see, it also in another spot. Thanks